encoding/gproperties/gproperties.go

```diff
@@ -0,0 +1,176 @@
+// Copyright GoFrame Author(https://goframe.org). All Rights Reserved.
+//
+// This Source Code Form is subject to the terms of the MIT License.
+// If a copy of the MIT was not distributed with this file,
+// You can obtain one at https://github.com/gogf/gf.
+
+// Package gproperties provides accessing and converting for .properties content.
+
```

Comment:
Do use empty line after package comment.

Reply:
deleted this empty line